html-and-the-web/html-and-the-web_div-salad/index.html

```diff
@@ -13,59 +13,59 @@
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
 	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
       <!--Which elements could be used to link to parts of the webpage,
 		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
+      <nav id="nav">
+        <a href="#about-me">About Me</a>
+        <a href="#contact">Contact</a>
+        <a href="#skills">Skillset</a>
+      </nav>
+    </header>
     <!--Looks like the following part is the central content of our webpage.-->
-    <div>
-      <div id="about-me">
-        <div>About me</div>
-        <div>
+    <main>
+      <article id="about-me">
+        <h2>About me</h2>
+        <p>
           Hi there! Welcome to my personal website. Here is gathered the most
           important information about me. If you want to work with me or get to
           know me, please don't hesitate to get in contact.
-        </div>
+        </p>
         <!--What element could be used to combine an image with a descriptive caption?-->
-        <div>
+        <section>
```

Comment:
For the image with a caption, you can swap the section tag with a figure tag and replace the p tag with figcaption. This will organize it so the text will appear just below the image!